docs/dsl.rst

```diff
@@ -10,18 +10,14 @@ On Statements
 On statements are used by :ref:`Reactors` wishing to make subscriptions to the :ref:`PowerPlant`.  Using this statement,
 developers can set the conditions under which desired :ref:`Reactions` will run.
 
-anatomy of an on statement
+Anatomy of an on statement
 --------------------------
 The on statement can be seen as containing three main parts.  The DSL Request, the Runtime Arguments, and the Callback.
 
-.. raw:: html
-
-    <font color="blue">On<...></font><font color="red">(Runtime, ... )</font><font color="green">.then(function);</font>
-    <br /><br />
-    <font color="blue">DSL Request</font><br />
-
-This is :red:`red !` And :blue:`this part is blue`.
+:blue:`On<Dsl, Word(s)>`\(:red:`Runtime, Args`).then(:green:`CallbackFunction`);
```

Comment:
If we want to use `rst`, we should actually use it.

Also, that escaped paren is deliberate. It's required after the backtick